app/views/transactions/new.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= render 'common/navigation', :title => 'Transaction' %>
+<section class="access flex">
+  <%= form_with model: operation, url: category_transactions_url(category_id: id), method: :post do |f| %>
```

Comment:
Please add a "Back" button (<) on `Add a new transaction` page so when the user clicks it, it navigates the user to the home page.